cns/logger/v2/cores/file.go

```diff
@@ -0,0 +1,53 @@
+package logger
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"gopkg.in/natefinch/lumberjack.v2"
+)
+
+type FileConfig struct {
+	Filepath   string        `json:"filepath"`
+	Level      string        `json:"level"`
+	level      zapcore.Level `json:"-"`
```

Comment:
these won't be _Marshalled_ correctly since I didn't write a custom Marshaller, but would we ever be going from struct -> text for Config?